### DISCUSSION TITLE
Probably this is the latest version

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#258] [Announcements]
Probably this is the latest version of the application and I hope that it will not be a lot of bugs and you will enjoy the application.

Unfortunately, I was never able to find ways to promote the project in the public. I wrote to bloggers, youtubers, and other famous people asking them to cover the app, write a review, or just tweet, but I never heard back. Perhaps I wrote to people not interested in it. Nevertheless, I think I have created a useful tool that is used 26k/month as of August 2022.

I hope in the future something to change. But for now I need to move on.

Special thanks to all those who have contributed or donated 🙏

You can always reach me by [email](mailto:reshetov.art@gmail.com).

Best regard,
[Anton](https://github.com/antonreshetov)

P.S. When it came for free with no effort, it has no value unfortunately. Wisdom that breaks my heart.

**Comment by tracure1337** [comment]
Thanks you for the effort. I really enjoy it.

**Reply by Faq** [reply]
Wow, sad news, just started to really use it. Thnx for Your time and effort to bring it to us.
